themes/jquery/css/docsearch.css

```diff
@@ -0,0 +1,19 @@
+/* Custom DocSeach CSS to adapt the generic one * See https://community.algolia.com/docsearch/styling.html for more info */
+nav#main .searchform {
+  text-shadow: none;
```

Comment:
This file still uses spaces, please change to tabs.

Reply:
My bad. fixed